code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -28,6 +28,7 @@ var/global/list/obj/item/device/pda/PDAs = list()
 	var/tnote = null //Current Texts
 	var/last_text //No text spamming
 	var/last_honk //Also no honk spamming that's bad too
+	var/last_trombone //yada yada spam

```

Comment:
Uh, why not use whatever spam var that the honk sound effect uses instead of making a new one?